package.json

```diff
@@ -10,7 +10,7 @@
     "build": "webpack --env=prod",
     "test": "npm-run-all --parallel --continue-on-error test-app test-api",
     "test-app": "karma start karma.conf.js",
-    "test-api": "./node_modules/.bin/mocha --recursive test/api",
+    "test-api": "./node_modules/.bin/mocha test/api --recursive --require test/api/api-test-globals",
```

Comment:
I don't think so. I've used it and I don't have mocha installed globally. That said, I'm not certain so it might be better to leave it the way it is